Dockerfile

```diff
@@ -174,7 +176,7 @@ RUN bundle exec rake assets:precompile
 ADD /server /opt/openstudio/server
 ADD .rubocop.yml /opt/openstudio/.rubocop.yml
 # Run bundle again, because if the user has a local Gemfile.lock it will have been overriden
-RUN bundle install
+RUN bundle update
```

Comment:
Are we sure we want to run update? I prefer bundle install so that it doesn't try to update any gems that we don't have locked down.